src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -10267,5 +10206,118 @@ public class Class1
                 Assert.False(state.IsAvailable);
             }
         }
+
+        [Fact]
+        [WorkItem(15532, "https://github.com/dotnet/roslyn/issues/15532")]
+        public async Task ExtractLocalFunctionCall()
+        {
+            var code = @"
+class C
+{
+    public static void Main()
+    {
+        void Local() { }
+        [|Local();|]
+    }
+}";
+            await TestExtractMethodAsync(code, @"
+class C
+{
+    public static void Main()
+    {
+        void Local() { }
+        NewMethod();
+    }
+
+    private static void NewMethod()
+    {
+        Local();
+    }
+}");
```

Comment:
That's an example with a Conflict-Marker in the source tree.  But warning should be supported as well:

https://github.com/dotnet/roslyn/blob/master/src/EditorFeatures/TestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs#L350